pkg/curatedpackages/reader.go

```diff
@@ -61,14 +87,51 @@ func fetchPackages(ctx context.Context, versionsBundle releasev1.VersionsBundle,
 	}
 	images := make([]releasev1.Image, 0, len(bundle.Spec.Packages))
 	for _, p := range bundle.Spec.Packages {
-		pI := releasev1.Image{
+		pHC := releasev1.Image{
 			Name:        p.Name,
 			Description: p.Name,
 			OS:          ctrl.OS,
 			OSName:      ctrl.OSName,
-			URI:         fmt.Sprintf("%s/%s:%s", defaultRegistry, p.Source.Repository, p.Source.Versions[0].Name),
+			URI:         fmt.Sprintf("%s/%s:%s", GetRegistry(ctrl.URI), p.Source.Repository, p.Source.Versions[0].Name),
```

Comment:
I got lost here. If we are doing some domain manipulation in the image mover, why is this necessary here?

Reply:
This is not doing any manipulation. It just extracts the image uri